libraries/Bluefruit52Lib/src/BLEAdvertising.cpp

```diff
@@ -385,6 +385,21 @@ bool BLEAdvertising::start(uint16_t timeout)
   return true;
 }
 
+// first set new advertising data with "Bluefruit.Advertising.setData(data,len) or "Bluefruit.ScanResponse.setData(data,len)"
+bool BLEAdvertising::updateData()
+{
+  // gap_adv long-live is required by SD v6
+  static ble_gap_adv_data_t gap_adv =
```

Comment:
this will create an extra memory for gap_adv, we should move it to class's member. Though I could do this myself